### PR TITLE
[Progress] Fix floating scrollbar

### PR DESCRIPTION
**Pull request by lfryemason**
Fixed:
* Two finger scroll not working
* Extra padding between table and scrollbar
* scrollbar only showing up after scroll event
* Loading/skeleton table showing scrollbar

## Links
[TEACH-925](https://codedotorg.atlassian.net/browse/TEACH-925)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/sectionProgressV2/ProgressTableV2.jsx [hunk 1]
```diff
@@ -86,45 +83,36 @@ function ProgressTableV2({
...
 
   const table = React.useMemo(() => {
-    const lessons =
-      isSkeleton && unitData === undefined
-        ? LESSON_SKELETON_DATA.map(id => ({id, isFake: true}))
-        : unitData?.lessons;
+    if (isSkeleton && unitData === undefined) {
+      const lessons = LESSON_SKELETON_DATA.map(id => ({id, isFake: true}));
+      return (
+        <div className={styles.tableLoading}>
```
Comment (etaderhold) [inline apps/src/templates/sectionProgressV2/ProgressTableV2.jsx:89]: No need for a floating scrollbar on the loading screen, I suppose.

File: apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx [hunk 2]
```diff
@@ -84,8 +74,10 @@ export default function FloatingScrollbar({
   }, [childContainerRef, scrollVisible, setScrollVisible]);
 
   React.useEffect(() => {
-    window.addEventListener('scroll', e => handleScrollAndResize(e));
-    window.addEventListener('resize', e => handleScrollAndResize(e));
+    window.addEventListener('scroll', () => handleScrollAndResize());
```
Comment (etaderhold) [inline apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx]: Do you need to wrap this function for some reason or would this work just as well?
```suggestion
    window.addEventListener('scroll', handleScrollAndResize);
```

File: apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx [hunk 3]
```diff
@@ -117,15 +103,23 @@ export default function FloatingScrollbar({
...
+      </div>
       <div
         className={classNames(
           styles.scrollBar,
           !scrollVisible && styles.bottomScrollBar
         )}
         onScroll={scrollChild}
         ref={scrollRef}
-        style={{width: childWidth + 'px'}}
+        style={{width: childWidth + 'px', height: scrollbarWidth + 'px'}}
```
Comment (etaderhold) [inline apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx]: There's a literal `20` up on line 68 of this file. Should it be replaced with `scrollbarWidth` too?

**Review by etaderhold — APPROVED** [review]

File: apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx (hunk 3 above)
Comment (lfryemason) [reply apps/src/templates/sectionProgressV2/floatingScrollbar/FloatingScrollbar.jsx]: Very good catch :)

**Pull request merged**